modules/standard/BigInteger.chpl

```diff
@@ -2967,15 +2967,15 @@ When ``n/d`` does not produce an integer, this method may produce incorrect resu
     var ret = this.probablyPrime(reps):int;
     return ret;
   }
- 
+  
+  enum primality {
```

Comment:
Since this is a new type that you are adding, let's add some documentation for it, to indicate its purpose.  See https://chapel-lang.org/docs/latest/tools/chpldoc/chpldoc.html#comment-style for the start of guidance on writing documentation comments.

Reply:
I have updated the docs, please check if the statements are fine.